cmd/jujud/updateseries/updateseries_test.go

```diff
@@ -266,76 +267,17 @@ func (s *updateSeriesCmdSuite) assertSetupAgentsForTest(c *gc.C) {
 	}
 }
 
-func (s *updateSeriesCmdSuite) TestFindAgents(c *gc.C) {
-	cmd := &UpdateSeriesCommand{toSeries: "xenial", fromSeries: "trusty", dataDir: s.dataDir}
-	err := cmd.findAgents(cmdtesting.Context(c))
-	c.Assert(err, jc.ErrorIsNil)
-
-	c.Assert(cmd.machineAgent, gc.Equals, s.machineName)
-	c.Assert(cmd.unitAgents, jc.SameContents, s.unitNames)
-}
-
-func (s *updateSeriesCmdSuite) TestFindAgentsFail(c *gc.C) {
-	agentsDir := path.Join(s.dataDir, "agents")
-	err := os.MkdirAll(path.Join(agentsDir, names.ApplicationTagKind+"-failme-0"), os.ModeDir|os.ModePerm)
-
-	cmd := &UpdateSeriesCommand{toSeries: "xenial", fromSeries: "trusty", dataDir: s.dataDir}
-	err = cmd.findAgents(cmdtesting.Context(c))
-	c.Assert(err, jc.ErrorIsNil)
-
-	c.Assert(cmd.machineAgent, gc.Equals, s.machineName)
-	c.Assert(cmd.unitAgents, jc.SameContents, s.unitNames)
-}
-
-func (s *updateSeriesCmdSuite) TestCreateAgentConf(c *gc.C) {
-	cmd := &UpdateSeriesCommand{toSeries: "xenial", fromSeries: "trusty", dataDir: s.dataDir}
-	conf, err := cmd.createAgentConf("machine-2")
-	c.Assert(err, jc.ErrorIsNil)
-	// Spot check Conf
-	c.Assert(conf.Desc, gc.Equals, "juju agent for machine-2")
-}
-
-func (s *updateSeriesCmdSuite) TestCreateAgentConfFailAgentKind(c *gc.C) {
-	cmd := &UpdateSeriesCommand{toSeries: "xenial", fromSeries: "trusty", dataDir: s.dataDir}
-	_, err := cmd.createAgentConf("application-fail")
-	c.Assert(err, gc.ErrorMatches, `agent "application-fail" is neither a machine nor a unit`)
-}
-
-func (s *updateSeriesCmdSuite) TestStartAllAgents(c *gc.C) {
-	cmd := &UpdateSeriesCommand{toSeries: "xenial", fromSeries: "trusty", dataDir: s.dataDir}
-	ctx := cmdtesting.Context(c)
-	err := cmd.findAgents(ctx)
-	c.Assert(err, jc.ErrorIsNil)
-	err = cmd.startAllAgents(ctx)
-	c.Assert(err, jc.ErrorIsNil)
-
-	s.assertServicesCalls(c, "Start", len(s.services))
-}
-
 func (s *updateSeriesCmdSuite) TestStartAllAgentsFailPreReboot(c *gc.C) {
-	s.PatchValue(&sysdIsRunning, func() (bool, error) { return false, nil })
+	s.manager = service.NewSystemdServiceManager(func() (bool, error) { return false, nil })
```

Comment:
Heather's right - although this creates an UpdateSeriesCommand, it doesn't actually use it, it's just testing manager.StartAllAgents. If there isn't already a test for this behaviour there can you move it to service/agentconf_test.go?